src/components/pages/donation_form/PaymentSummarySection.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+	<div>
+		<h3>{{ $t('donation_form_summary_bank_details') }}</h3>
+		<strong>{{ $t('donation_form_summary_iban') }}</strong> {{ bankData.iban }}<br>
+		<strong v-if="bankData.bic">{{ $t('donation_form_summary_bic') }}</strong> {{ bankData.bic }}<br>
+		<strong v-if="bankData.bankName">{{ $t('donation_form_summary_bank_name') }}</strong> {{ bankData.bankName }}
```

Comment:
```suggestion
		<ul class="bankdata">
				<li><strong>{{ $t('donation_form_summary_iban') }}</strong> {{ bankData.iban }}</li>
				<li v-if="bankData.bic"><strong>{{ $t('donation_form_summary_bic') }}</strong> {{ bankData.bic }}</li>
				<li v-if="bankData.bankName"><strong>{{ $t('donation_form_summary_bank_name') }}</strong> {{ bankData.bankName }}</li>
		</ul>
```

You'll need to add stylings to the `summary.scss` for the list.